### ISSUE TITLE
Janhvi Mandali

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#172]

**Comment by aflies1** [comment]
Career? 
Her projects have given her experience in R, Python, Tableau, excel
Bach. in Elec Eng. back in India
Internship on the SQL side
Has some experience as a mentor for incoling students, curreiculm and coursework
Looking to have a role that involves data, something with Data excites her
Industry? Retail and consulting is of most interest

Python?
Was the 4th language she learned in high school
Has used over the last years in grad school
3.5/5 self score
Does feel comfortable and is able to pick new items as well

Large Data sets?
Had 8K rows with 10-12 variables
Idea to predict what ingredients in Milk make it sell?
Everyone is purchasing and its the main item purchased in a grocery score
Diets have changed and other reasons why customers have changed what milk they buy and why
Created some predictive models within this project
Would have changed some items or more data to secure customer

Goldman
Did work with Json and Elastic

Schedule
40 hours, Will be taking 9 credits
All start in the evening (7pm CT)
CPT?
No Classes next semester

ML?
Python has been within her coursework

**Comment by aflies1** [comment]
Resume; https://drive.google.com/drive/folders/1nWwkVJCtvCkvgcx3xfonrgqEFwSh67dO

**Comment by ldaudet** [comment]
Janhvi Mandali

she is currently getting her master of science in business analytics at dallas, after her bachelor in electronics in India.

She always wanted to do business, but wanted to make sure to not do just that. While in school, she figured out that CS and tech was an interest of hers, and well connected to business. With a good understanding of data comes a good understanding of business.

She talks fast, with a slight accent that isn't detrimental for understanding at all.

She has experience in EDA, and data cleanup and analysis. 

She has one project that she has done in Python. She was looking at differnt types of Cereals to show what component participate to sales of sadi cereals.

She is also about to start a couple of more projects in python. She says that she is pretty confident in python, thgouh of her own admition not expert by any means. 

ML expertise is still very nascent, as she is going through the classes right now. Her most reliable language is R, for having most experience in it.

While at goldman sachs, she had to send and retrieve data from an elastic search cluster.

She seem very professional, asking pertinent questions and being able to think on her feet. With a bit more experience, she would be great on a project.


TL;DR: Very good head on her shoulders, but I think too green still to really be a part of a project
